demo/index.html

```diff
@@ -73,14 +73,9 @@
         --paper-icon-button-ink-color: var(--paper-indigo-500);
       }
 
-      paper-icon-button.huge::shadow #icon {
-        width: 100px;
-        height: 100px;
-      }
-
-      paper-icon-button.huge #icon {
-        width: 100px;
-        height: 100px;
+      paper-icon-button.huge {
+        --iron-icon-height: 100px;
+        --iron-icon-width: 100px;
```

Comment:
Working for me inside `custom-style` as per above http://jsbin.com/pojatemobe/edit?html,output but afaik it should also work inside `:root`.